### DISCUSSION TITLE
Help me find the right gear

### DISCUSSION THREAD

**Discussion by edisax** [bitcraze/discussions#364] [Q&A]
Hi! 
Im interested in creating a similar shows to these two:

1) https://www.youtube.com/watch?v=2MhiD7Hy-AM

2)https://www.youtube.com/watch?v=Cg6V3qrSguk&list=PLj9XMmQVSr-AC2FHF-CGIv6ZaxsL_L75N

Can you please guide me on what gear and software I need to get in order to get started :D 
Thanks in advance!

**Comment by tobbeanton** [comment]
Even though our main focus is a flying development platform you should be able to build something similar with a [lighthouse swarm bundle](https://store.bitcraze.io/collections/bundles/products/lighthouse-swarm-bundle), added [led-rings](https://store.bitcraze.io/collections/decks/products/led-ring-deck) and [skybrush](https://skybrush.io/). I would also recommend the [thrust upgrade kit](https://www.bitcraze.io/2022/10/thrust-upgrade-kit-for-the-crazyflie-2-1/) for better flight stability that should be available in mid November.

**Reply by edisax** [reply]
Thanks your reply and tips!
Can you by any chance understand what kind of tool they have attached to their hand/wrist to control the drones movement with?

**Reply by knmcguire** [reply]
The first video I can't see what he is using. Perhaps good time chrography or an external camera that can recognize hand gestures?
Probably the second video uses something more closer to https://hi5vrglove.com/

**Reply by edisax** [reply]
Thanks a lot for the reply and suggestions!

Found another one where I think they're using crazyflies. My question is can I achieve a similar autonomous fly show using the AI Bundle?
Does every crazyflie need the ai-deck or is it just needed in one of the swarms?
https://youtu.be/RCXGpEmFbOw?t=491

**Reply by tobbeanton** [reply]
That show is using the Crazyflie and their own UWB system, similar to our [Loco positioning system](https://www.bitcraze.io/documentation/system/positioning/loco-positioning-system/). They are not using the AI-deck. Doing a show is still very complex and it is very hard to get everything right and reliable.
